src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/ExpectationsRouteConfig.kt

```diff
@@ -0,0 +1,20 @@
+package org.vaccineimpact.api.app.app_start.route_config
+
+import org.vaccineimpact.api.app.app_start.*
+import org.vaccineimpact.api.app.controllers.ExpectationsController
+
+object ExpectationsRouteConfig : RouteConfig
+{
+    private val getAllPermissions = setOf(
+        "*/responsibilities.read"
```

Comment:
is this the right permission to require?

Reply:
This seems right to me